README.md

```diff
@@ -129,11 +131,12 @@ imports:
     # apply
     vendor/bin/rector process src
     ```
+<br>
+<br>
 
+## Extra Features
 
-## Features
-
-### Extra Autoloading
+### More Autoloading
```

Comment:
Why "More" ?

Reply:
Because you had `Extra Features` and you don't want to have `Extra Features` and `Extra Autoloading` right underneath it, so just used a synonym

Reply:
@Kerrialn I also tried to improve the README some time ago, maybe you can find some inspiration there: https://github.com/rectorphp/rector/pull/2116/files#

e.g. 
- group the configuration together (https://github.com/rectorphp/rector/pull/2116/files#diff-04c6e90faac2675aa89e2176d2eec7d8R134)
- re-write the headline for the "Coding Standards" part: https://github.com/rectorphp/rector/pull/2116/files#diff-04c6e90faac2675aa89e2176d2eec7d8R116

Reply:
Tip: start with small PRs, one at a time, they're easier to review and accept. E.g. if this one is split into 5 PRs, 4 of them are accepted and one rejeted. Job done :)

In time those PRs build trust and in the end, you can send huge changes without much prolonging. At least at this state of project we are now.